src/_pytest/doctest.py

```diff
@@ -120,32 +121,32 @@ def pytest_unconfigure() -> None:
 
 
 def pytest_collect_file(
-    path: py.path.local, parent: Collector,
+    fspath: Path, path: py.path.local, parent: Collector,
```

Comment:
We don't use `path` anymore, we can drop it from the arguments.

```suggestion
    fspath: Path, parent: Collector
```

Reply:
It's actually still used, just well hidden with the `path`/`fspath` naming mixing up.